rxlifecycle/src/main/java/com/trello/rxlifecycle/ActivityLifecycleProvider.java

```diff
@@ -8,7 +8,7 @@
  * Common interface for all RxActivity extensions.
  *
  * Useful if you are writing utilities on top of rxlifecycle-components,
- * or you are implementing your own component not supported in this library.
+ * or if you are implementing your own component which is not supported in this library.
```

Comment:
Yes though I had the feeling it was a bit weird to read. I changed it now to your proposed change.